### PR TITLE
Fixed put.group_hunt_group method

### PR DESCRIPTION
**Pull request by LivCurtis**
Assigned an empty dict to "serviceInstanceProfile" as this needs to exist for the method to work. Also corrected a couple of things in the docstring.

### CODE REVIEW

File: odins_spear/methods/put.py [hunk 1]
```diff
@@ -897,10 +897,10 @@ def group_hunt_groups_status(self, hunt_group_user_ids: list, status: bool =True
         
     
     def group_hunt_group(self, service_provider_id: str, group_id: str, hunt_group_user_id: str, updates: dict):
```
Comment (Jordan-Prescott) [inline odins_spear/methods/put.py:899]: Change hunt_group_user_id to service_user_id in these args.

File: odins_spear/methods/put.py [hunk 2]
```diff
@@ -913,7 +913,8 @@ def group_hunt_group(self, service_provider_id: str, group_id: str, hunt_group_u
         
         updates["serviceProviderId"] = service_provider_id
         updates["groupId"] = group_id
-        updates["serviceUserId"] = hunt_group_user_id          
+        updates["serviceUserId"] = hunt_group_user_id
```
Comment (Jordan-Prescott) [inline odins_spear/methods/put.py:916]: Should match new argument

File: odins_spear/methods/put.py [hunk 3]
```diff
@@ -913,7 +913,8 @@ def group_hunt_group(self, service_provider_id: str, group_id: str, hunt_group_u
...
+        updates["serviceInstanceProfile"] = {}
```
Comment (Jordan-Prescott) [inline odins_spear/methods/put.py:917]: Check if this is in the updates otherwise here you will override it

**Review by Jordan-Prescott — CHANGES_REQUESTED** [review]
Adjust for comments please.

File: odins_spear/methods/put.py (hunk 1 above)
Comment (LivCurtis) [reply odins_spear/methods/put.py]: Done :)

File: odins_spear/methods/put.py (hunk 2 above)
Comment (LivCurtis) [reply odins_spear/methods/put.py]: Done :)

File: odins_spear/methods/put.py (hunk 3 above)
Comment (LivCurtis) [reply odins_spear/methods/put.py]: Done :)

**Review by LivCurtis — COMMENTED** [review]
Have made the suggested changes.

**Review by Jordan-Prescott — APPROVED** [review]
All looks good, nice work Liv!

**Pull request merged**